addons/mail/controllers/discuss/binary.py

```diff
@@ -86,18 +106,18 @@ def discuss_channel_attachment(self, channel_id, attachment_id, download=None):
     )
     def fetch_image(self, channel_id, attachment_id, width=0, height=0, **kwargs):
         channel_member_sudo = request.env["discuss.channel.member"]._get_as_sudo_from_request_or_raise(
-            request=request, channel_id=int(channel_id)
+            request=request, channel_id=channel_id
         )
         domain = [
-            ("id", "=", int(attachment_id)),
-            ("res_id", "=", int(channel_id)),
+            ("id", "=", attachment_id),
+            ("res_id", "=", channel_id),
             ("res_model", "=", "discuss.channel"),
         ]
         attachment_sudo = channel_member_sudo.env["ir.attachment"].search(domain, limit=1)
         if not attachment_sudo:
             raise NotFound()
         return (
             request.env["ir.binary"]
-            ._get_image_stream_from(attachment_sudo, width=int(width), height=int(height))
+            ._get_image_stream_from(attachment_sudo, width=width, height=height)
```

Comment:
Why the diff here? Removing the int cast is making the route crash.

Reply:
the route is specifying the type already so it should be fine to remove the cast 
`
            "/discuss/channel/<int:channel_id>/image/<int:attachment_id>",
            "/discuss/channel/<int:channel_id>/image/<int:attachment_id>/<int:width>x<int:height>",
            `

Reply:
size params are not specified when when using the first version of the route and passing them as kwargs